package.json

```diff
@@ -134,7 +134,7 @@
     "prettier": "^1.18.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "^4.0.0",
+    "react-scripts": "^3.4.3",
```

Comment:
Blocker - we lose things like React Fast Refresh with this downgrade.
We have full control of our linter. Fix the specific issue, preferably in another PR.
No need for this to get in the way of the SQFormMultiValue PR